.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
     - DJANGO="django>=1.11,<2"
     - DJANGO="django>=2.0,<2.1"
     - DJANGO="django>=2.1,<2.2"
+    - DJANGO="django>=1.11.17,<2.2"
```

Comment:
Not sure what you are testing with this line. The latest 2.1? That's already covered in the previous line.

Reply:
I feel like I need it for py3.7, which is 1.11.17 and higher.